bin/tigmint-make-try

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/make -rRf
```

Comment:
Is this file just a copy of tigmint-make? Probably don't want to merge this into master.